docs/index.rst

```diff
@@ -1,34 +1,96 @@
-Flocker Documentation
-=====================
+.. raw:: html
 
-Welcome to the Flocker docs! If you're just getting started with Flocker, it might be worth reading our :ref:`introduction`, or trying our simple tutorial on the `Getting Started with Flocker`_ page.
+    <style>
+        .toctree-wrapper { display:none; }
+    </style>
 
-In the documentation listed below, you'll be able to find instructions on how to install, configure, and use Flocker.
-You will also find instructions on how to install and use :ref:`docker-plugin`.
+==================
+Installing Flocker
+==================
 
-You'll be able to find concept and reference material, and our :ref:`Labs section<labs-projects>` contains all the information you need about our experimental projects.
+.. _supported-orchestration-frameworks:
 
-Flocker is an open-source project, and in our :ref:`Getting Involved section<getting-involved>`, there is information about how you can collaborate with ClusterHQ.
+Flocker allows you to launch and move stateful containers by integrating with a cluster manager of your choice.
+
+To install Flocker, first choose which cluster manager you are using, or intend to use:
+
+.. raw:: html
+
+	<!-- This too needs to become Sphinx directives, rather than raw HTML. -->
+	<div class="pods-eq">
+	    <div class="pod-boxout pod-boxout--orchestration pod-boxout--recommended">
+			<img src="_images/docker2x.png" alt="Docker logo"/>
+			<span>Docker, Swarm, Compose <em>Fastest</em></span>
```

Comment:
"Docker, Swarm, Compose" sounds more like a stack than a "cluster manager".
Perhaps better to say "Docker Swarm, with Docker Compose."